### DISCUSSION TITLE
Does deepxde use actually a solver when the problem's solution is not known?

### DISCUSSION THREAD

**Discussion by theo-ifaistos** [lululxvi/deepxde#930] [Q&A]
Hello! 

I got confused when I was looking at the documentation of the class "deepxde.data.pde.PDE" and its time dependent counterpart. 
Does this class actually use a solver for the relevant governing equations in real time during the training phase? And if so which one?

I looked at the source code and I couldn't figure it out..! Any help would be much appreciated!

Thanks for the awesome library,

Kind regards!

**Comment by lululxvi** [comment]
No, See https://doi.org/10.1137/19M1274067
